### DISCUSSION TITLE
How to install Jupyter/widgets

### DISCUSSION THREAD

**Discussion by DonJayamanne** [yitzchak/common-lisp-jupyter#93] [Q&A] [answered]
Hi, I'm new to lisp hence this is a newbie question.
I've installed roswell and ran `ros install common-lisp-jupyter` . I can now launch Jupyter lab and run some sample lisp (from the samples provided).
However I'm unable to get the widgets working, I'm assuming this is because I need to install the `jupyter/widgets` package.
Please could you advice how I could do this.
I've also tried the following without any success:
```
> ros run
* (ql:quickload :jupyter/widgets)
```

Thanks

**Comment by DonJayamanne** [comment]
Never mind, its working, not sure what i did though.

**Comment by yitzchak** [comment, marked as answer]
The widgets package is included. You shouldn't need to do anything.
